cyto_dl/nn/vits/utils.py

```diff
@@ -40,3 +47,8 @@ def get_positional_embedding(
             cls_token = torch.zeros(1, 1, emb_dim)
             pe = torch.cat([cls_token, pe], dim=0)
         return torch.nn.Parameter(pe, requires_grad=False)
+
+
+def validate_spatial_dims(spatial_dims, tuples):
```

Comment:
I feel like the code might be clearer by not having this be a separate function and just calling these 2 lines in every class? I thought this function was doing a lot more based on the name (like some math to check that the spatial dimensions of each patch and mask is correct). What do you think?

Reply:
I'd prefer renaming it to something clearer rather than repeating the code, maybe `match_tuple_to_spatial_dims`?

Reply:
sounds good